### DISCUSSION TITLE
Get-er for geneScoreMatirx?

### DISCUSSION THREAD

**Discussion by Zepeng-Mu** [GreenleafLab/ArchR#729] [Questions / Documentation] [answered]
Hi, I'm wondering whether there is a getter function for GeneScoreMatrix. I did not find it in the `Functions` page in GitHub website.
Thanks!

**Comment by Zepeng-Mu** [comment]
Seems that I can try `.getMatrixValues`.

**Comment by rcorces** [comment, marked as answer]
https://www.archrproject.com/reference/getMatrixFromProject.html
